ush/templates/FV3LAM_wflow.xml

```diff
@@ -320,6 +320,8 @@ MODULES_RUN_TASK_FP script.
     <dependency>
       {%- if machine in ["WCOSS_DELL_P3", "WCOSS_CRAY"]  %}
       <datadep age="00:00:05:00"><cyclestr offset="-{{ extrn_mdl_ics_offset_hrs }}:00:00">{{ extrn_mdl_sysbasedir_ics }}/gfs.@Y@m@d/@H/atmos/gfs.t@Hz.pgrb2.0p25.f{{ "%03d" % extrn_mdl_ics_offset_hrs }}</cyclestr></datadep>
+      {%- elif machine in ["ORION"]  %}
+      <datadep age="00:00:05:00"><cyclestr offset="-{{ extrn_mdl_ics_offset_hrs }}:00:00">{{ extrn_mdl_sysbasedir_ics }}/gdas.@Y@m@d/@H/atmos/gdas.t@Hz.atmf{{ "%03d" % extrn_mdl_ics_offset_hrs }}.nc</cyclestr></datadep>
       {% else %}
```

Comment:
The file name really should be stored as either an entity, or passed in as a template field so that we don't have to repeat the entire line. That's a lot additional information here. 

Also this is a very convoluted bit of logic here that indicates that you have to choose GDAS input on Orion. Those are two totally independent choices, and surely won't always be the case.

Reply:
Right now, I don't have a good way to setup those dependence except for make them depend on machine type. We need to refactor both get_extern tasks and xml dependences to make more system more portable.

Reply:
You can still have the logic in the setup or generate script, and ensure that the logic is explained clearly. The only thing that differs between the three long repeated lines is the name of the file, so it would be best to template the filename rather than repeating all the information in the if statement logic.

Reply:
This is not simple effort can be done in this PR. We need to talk with DTC to get a better solution to get and check external.